composer.json

```diff
@@ -4,8 +4,7 @@
     "keywords": ["Mockup"],
     "require": {
         "php": ">=5.4.1",
-        "symfony/symfony": "~2.0",
-        "claroline/kernel-bundle": "~6.0"
+        "claroline/distribution": "~7.0"
```

Comment:
Can you change the version constraint to `>=7.0`? Otherwise that won't work with the master branch of claroline/Distribution.